src/components/views/messages/TextualBody.js

```diff
@@ -72,12 +73,16 @@ module.exports = React.createClass({
         textArea.value = text;
         document.body.appendChild(textArea);
         textArea.select();
+
+        let successful;
```

Comment:
` = false` here otherwise you'll return `undefined` on failure which will be a bit confusing.

Reply:
Done